### PR TITLE
filterStrategy, procedure EntryElement change, FamilyHistory typo fix

### PR DESCRIPTION
**Pull request by mlterryMitre**

### CODE REVIEW

File: spec/ig-mcode-config.json [hunk 1]
```diff
@@ -14,16 +14,34 @@
...
+            "primary": ["mcode",
+                "shr.entity.Patient",
+                "shr.entity.Person",
+                "shr.entity.Practitioner",
+                "shr.medication.MedicationStatement",
+                "shr.medication.MedicationAdministered",
+                "shr.procedure.LaboratoryProcedurePerformed",
+                "shr.procedure.SurgicalProcedurePerformed",
+                "shr.procedure.DiagnosticImagingPerformed",
+                "shr.onclogy.HER2ReceptorStatus"],
```
Comment (rutemartins) [inline spec/ig-mcode-config.json]: there's a typo here. Should be shr.oncology. On that note, why are we including HER2ReceptorStatus? Testing purposes only?

File: spec/ig-mcode-config.json [hunk 2]
```diff
@@ -14,16 +14,34 @@
...
+        "target": ["mcode",
+            "shr.entity.Patient",
+            "shr.entity.Person",
+            "shr.entity.Practitioner",
+            "shr.medication.MedicationStatement",
+            "shr.medication.MedicationAdministered",
+            "shr.procedure.LaboratoryProcedurePerformed",
+            "shr.procedure.SurgicalProcedurePerformed",
+            "shr.procedure.DiagnosticImagingPerformed",
+            "shr.onclogy.HER2ReceptorStatus"]
```
Comment (rutemartins) [inline spec/ig-mcode-config.json]: Same as above

File: spec/ig-mcode-config.json (hunk 1 above)
Comment (markkramerus) [reply spec/ig-mcode-config.json]: Actually, there is no more shr.oncology. I changed it to brca because the namespace is really breast cancer specific.

Comment (rutemartins) [reply spec/ig-mcode-config.json]: Hmm... Seems like the filter should throw some errors when we reference non-existing namespaces or elements.

Comment (mlterryMitre) [reply spec/ig-mcode-config.json]: testing purposes.

**Pull request merged**